pallets/eth-bridge/src/types.rs

```diff
@@ -52,6 +53,22 @@ impl Identifiable for LowerProofRequestData {
         return self.id
     }
 }
+
+// Data related to generating confirmations
+#[derive(Encode, Decode, Clone, PartialEq, Eq, Debug, Default, TypeInfo, MaxEncodedLen)]
+pub struct ActiveConfirmation {
+    pub msg_hash: H256,
+    pub confirmations: BoundedVec<ecdsa::Signature, ConfirmationsLimit>,
+}
+
+// Persisten storage struct to hold lower proof that can be claimed on ethereum
+#[derive(Encode, Decode, Clone, PartialEq, Eq, Debug, Default, TypeInfo, MaxEncodedLen)]
+pub struct LowerProofData {
+    pub params: BoundedVec<(BoundedVec<u8, TypeLimit>, BoundedVec<u8, ValueLimit>), ParamsLimit>,
+    pub lower_data: BoundedVec<u8, LowerDataLimit>,
+    pub is_claimed: Option<bool>,
```

Comment:
What's this field for?

Reply:
this was a field to close the loop but after our discussion, if/when we get confirmation from Ethereum, we will remove the entry instead of updating a field. So will remove this field.